pkgs/tools/misc/pridefetch/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub, lib, python39, zip }: let
+  version = "1.0.0";
+  sha256 = "sha256-/o4er8bO/3HUFXzP+sC+5DYv9EwmxW05o1RT5fEulEg=";
+
+  name = "pridefetch";
+  src = fetchFromGitHub {
+    owner = "SpyHoodle";
+    repo = name;
+    rev = "v" + version;
+    inherit sha256;
+  };
+in stdenv.mkDerivation {
```

Comment:
Could `buildPython*` be used here maybe?

Reply:
I don't think so, as pridefetch doesn't have stuff needed for a python package such as `setup.py`. It's an idea for the future, but it wouldn't be able to be used on the current version